### PR TITLE
Libraries are now versioned

### PR DESCRIPTION
**Pull request by DavidPoliakoff**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,11 @@
 option(GOTCHA_ENABLE_TESTS "Enable internal tests" Off)
 option(GOTCHA_ENABLE_COVERAGE_TESTS "Enable coverage tests" Off)
 project(gotcha)
+
+set(LIBTOOL_INTERFACE 0)
```
Comment (DavidPoliakoff) [inline CMakeLists.txt]: @mplegendre : for managing versioning, our new thing we have to handle is to maintain this triple appropriately as we make PRs

**Comment by mplegendre** [comment]
Commits look good, though travis isn't completing for some reason.

Recommend adding an open issue of "package gotcha for release 0.1", where we add notes like "update library version" and any other TODOs we come across.

**Comment by DavidPoliakoff** [comment]
https://github.com/LLNL/GOTCHA/projects/1 I'll try this as a project, haven't used these on GH before.

**Comment by DavidPoliakoff** [comment]
Also I think Travis is just way behind right now

**Pull request merged**